tools/dissector/packet-opendds.h

```diff
@@ -14,13 +14,16 @@ extern "C" {
 
 #include "config.h"
 
+
 #include <glib.h>
 #include <gmodule.h>
 
 #include <epan/value_string.h>
 #include <epan/ipproto.h>
 #include <epan/packet.h>
 #include <epan/dissectors/packet-tcp.h>
+#include <epan/expert.h>
+#include <wsutil/report_message.h>
```

Comment:
Using wsutil/report_message.h break the build for Wireshark 2.2 (used in ubuntu 16.04)
The header report_err.h has been renamed to report_message.h with Wireshark 2.4 release.
https://github.com/wireshark/wireshark/commit/2c44a7f930967335b442e899c17e1eb46831e3c1

Reply:
Alright, the signature of the function I was using (`report_failure`) didn't change so I set it to include report_message if using 2.4 or greater, else report_err.